app/src/main/java/org/fossasia/susi/ai/login/LoginPresenter.kt

```diff
@@ -128,4 +128,4 @@ class LoginPresenter(loginActivity: LoginActivity): ILoginPresenter, ILoginModel
     override fun onDetach() {
         loginView = null
     }
-}
+}
```

Comment:
It is a good practice to have a newline at the end of the file. Not compulsory but a suggestion.